studies/migrations/0064_auto_20200429_1433.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.1 on 2020-04-29 18:33
```

Comment:
What is this migration generated from? Was it manually invoked?

If so, could you migrate back one and re-run the makemigration command with `--name=whatever` so we know what it's about?

Reply:
Done, sort of. This one was generated when switching to Django 2.0 but there are no accompanying model changes, so I just named it based on that upgrade. Similarly for the studies migrations (there were originally one set each for 1.11 -> 2.0, 2.0 -> 2.1). If these names should be more descriptive about the relevant changes that are being imposed I can change them, the Django version seemed more practically relevant for us knowing when these need to be applied.